inst/templates/call-connect-publish.yml

```diff
@@ -0,0 +1,36 @@
+# This action publishes to connect using the git-backed workflow. Note that
+# it is only necessary to do it this way if the product is in a private or
+# internal repository; otherwise, it is far easier to deploy following the
+# instructions in https://docs.posit.co/connect/user/git-backed/
+
+# To use this workflow, the user will need to generate the manifest.json, create
+# and api key from connect, and add 2 secrets, one named

```

Comment:
Suggestion: maybe delete "create an api key from connect" because "add 2 secrets" implies that users need to create an api key from connect.